cmd/pbackend/api.go

```diff
@@ -71,7 +71,10 @@ func api() {
 		if err != nil {
 			logger.Fatal().Err(err).Msg("Unable to initialize the platform kafka")
 		}
-		notifications.Initialize(ctx)
```

Comment:
Shall we drop the `Initialize` function then?

Reply:
Sorry what? I just wrapped it with an if-statement.

Reply:
Wow, I should learn how to read :face_in_clouds: